### PR TITLE
Create vi-VN

### PR DESCRIPTION
**Pull request by yakitovn**

### CODE REVIEW

**Review by lizheming — APPROVED** [review]

File: packages/client/src/config/i18n/vi-VN.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+export default generateLocale([
+  'Tên',
+  'Tên không được nhỏ hơn 3 ký tự.',
+  'E-Mail',
+  'Vui lòng xác nhập địa chỉ email của bạn.',
+  'Website',
+  'Tùy chọn',
+  'Hãy bình luận có văn hoá!',
+  'Chưa có bình luận',
+  'Gửi,
```
Comment (Mister-Hope) [inline packages/client/src/config/i18n/vi-VN.ts:12]: Missing quote

File: packages/client/src/config/i18n/vi-VN.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  'Trả lời',
+  'Hủy bỏ',
+  'bình luận',
+  'Làm mới',
+  'Tải thêm...',
+  'Xem trước',
+  'Emoji',
+  'Tải lên hình ảnh',
+  'giây trước',
+  'phút trước,
```
Comment (Mister-Hope) [inline packages/client/src/config/i18n/vi-VN.ts:22]: Missing quote

File: packages/client/src/config/i18n/vi-VN.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  'giờ trước,
```
Comment (Mister-Hope) [inline packages/client/src/config/i18n/vi-VN.ts:23]: Missing quote

File: packages/client/src/config/i18n/vi-VN.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  'ngày trước,
```
Comment (Mister-Hope) [inline packages/client/src/config/i18n/vi-VN.ts:24]: Missing quote

File: packages/client/src/config/i18n/vi-VN.ts [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  'Vừa xong,
```
Comment (Mister-Hope) [inline packages/client/src/config/i18n/vi-VN.ts:25]: Missing quote

File: packages/client/src/config/i18n/vi-VN.ts [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+  'Đang tải lên',
+  'Đăng nhập',
+  'đăng xuất',
+  'Admin',
+  'Sticky',
+  'từ',
+  'Please input comments between $0 and $1 words!\n Current word number: $2',
```
Comment (Mister-Hope) [inline packages/client/src/config/i18n/vi-VN.ts:32]: Missing translation.

**Review by Mister-Hope — COMMENTED** [review]
Please open a new PR and make sure these are fixed.

**Comment by Mister-Hope** [comment]
BTW, there are some description why am I reverting this PR in https://github.com/walinejs/waline/pull/940. We still weclcome you to add this language.

**Pull request merged**